server/cmake/modules/FindLibClang.cmake

```diff
@@ -59,6 +61,9 @@ foreach (version ${LIBCLANG_KNOWN_LLVM_VERSIONS})
     "/usr/local/lib/llvm-${version}/lib"
     # FreeBSD ports versions
     "/usr/local/llvm${undotted_version}/lib"
+    # Gentoo clang-4
+    "/usr/lib/llvm/${version}/lib"
+    "/usr/lib/llvm/${version}/lib64"
```

Comment:
Do you need to specify lib64? I think `cmake` handles it automatically, c.f:

> If the global property FIND_LIBRARY_USE_LIB64_PATHS is set all search paths will be tested as normal, with “64/” appended, and with all matches of “lib/” replaced with “lib64/”. This property is automatically set for the platforms that are known to need it if at least one of the languages supported by the PROJECT command is enabled.

-- https://cmake.org/cmake/help/v3.1/command/find_library.html?highlight=lib64